zhaquirks/tuya/tuya_smoke.py

```diff
@@ -60,8 +64,22 @@ def _update_attribute(self, attrid, value):
     .applies_to("_TZE284_rccxox8p", "TS0601")
     .applies_to("_TZE200_vzekyi4c", "TS0601")
     .applies_to("_TZE204_vawy74yh", "TS0601")
+    .tuya_smoke(dp_id=1)
+    .skip_configuration()
+    .add_to_registry()
+)
+
+(
+    TuyaQuirkBuilder("TZE200_0zaf1cr8", "TS0601")
     .applies_to("_TZE284_0zaf1cr8", "TS0601")
     .tuya_smoke(dp_id=1)
+    .tuya_binary_sensor(
+        dp_id=14,
+        attribute_name="battery_low",
+        translation_key="battery_low",
+        fallback_name="Battery low",
+    )
```

Comment:
There's `BinarySensorDeviceClass.BATTERY` where "On means low, Off means normal". We might wanna use that.

Reply:
Nice, will update to use that.